eva_submission/vep_utils.py

```diff
@@ -131,8 +131,14 @@ def get_vep_cache_version_from_ftp(assembly_accession, ensembl_assembly_name=Non
 @retry(tries=4, delay=2, backoff=1.2, jitter=(1, 3))
 def get_species_and_assembly(assembly_acc):
     """
-    Returns Ensembl species name, the assembly name of the provided assembly and if the accession is the currently
-     supported assembly, or None if any aren't found.
+    For the provided assemlbly, search for the assembly name and the associated species name in Ensembl (via the
+    taxonomy of the assembly).
+    This function return the species name associated with the supported assembly when several assemblies are supported
+    in the current version.
+    When the assembly is not supported return the species name marked as "reference"
+    In some cases, ensembl has several names for that species (because it supports
+    multiple strains) so will return the name associated with this reference
```

Comment:
```suggestion
    This function returns the species name associated with the provided assembly accession if it is supported
    in the current version of Ensembl.
    If the assembly is not supported, returns the species name marked as "reference" among the strains that
    Ensembl does support.
```

Tried to make this clearer and more concise but feel free to ignore if it's not quite right...